app/models/welcome_decorator.rb

```diff
@@ -0,0 +1,26 @@
+class WelcomeDecorator
+  attr_reader :show
+
+  def initialize(user)
+    @show = Github.new(user)
+  end
+
+  def current_streak
+    show.current_streak
+  end
+
+  def longest_streak
+    show.longest_streak
+  end
+  def year_to_date
```

Comment:
Use empty lines between method definitions.<br>Use `delegate` to define delegations.